client/state/stats/lists/utils.js

```diff
@@ -84,11 +84,12 @@ export function parseAvatar( avatarUrl ) {
 
 /**
  * Builds data into escaped array for CSV export
- * @param   {Object} data   Normalized stats data object
- * @param   {string} parent Label of parent
- * @returns {Array}         CSV Row
+ * @param   {Object} data                                               Normalized stats data object
+ * @param   {(value: unknown[], data?: Object) => unknown[]} modifierFn Modifies the export row.
+ * @param   {string} parent                                             Label of parent
+ * @returns {Array}                                                     CSV Row
  */
-export function buildExportArray( data, parent = null ) {
+export function buildExportArray( data, modifierFn = null, parent = null ) {
```

Comment:
```suggestion
export function buildExportArray( data, parent = null, modifierFn = null ) {
```
Could we move the newly added argument to the end to avoid potentially breaking the existing usage?

Reply:
Yep, fixed in commit https://github.com/Automattic/wp-calypso/pull/103581/commits/cb4aaba70ab05d4350819d83fb9abb9da1389f2d.